cmd/ui.go

```diff
@@ -133,6 +151,7 @@ func (ui *UI) PrintMetrics() string {
 		"Comments",
 		"Participants",
 		"Feature Lead Time",
+		"First Review to Last Review",
```

Comment:
Can you please add a note that describes this in the `README` under **Metric definitions**?